public/request/leaderboard/remove-entry.php

```diff
@@ -3,7 +3,10 @@
 require_once __DIR__ . '/../../../vendor/autoload.php';
 
 $leaderboardId = requestInput('l', 0, 'integer');
-$targetUser = requestInput('t');
+$targetEntry = explode(",", requestInput('t'));
+$targetUser = $targetEntry[0];
+$targetUserScore = $targetEntry[1];
```

Comment:
Does the user have to be passed in via query parameter?
`removeLeaderboardEntry` could return the deleted entry as well which could contain the score.

Reply:
This was just the way I had thought about going about it. I've switched it over to have removeLeaderboardEntry now so hopefully that looks better but let me know if I need to make more adjustments.